<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewGui</class>
 <widget class="QMainWindow" name="ViewGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>753</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>8000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ViewGui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="top">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>651</y>
      <width>931</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="bottom">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>238</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid Black;
    border-radius: 9px;
    margin-top: 0.5em;
background-color : white;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="menu">
     <item>
      <widget class="QPushButton" name="start">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#start
    {
        background-color: red
    }</string>
       </property>
       <property name="text">
        <string>START</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="choice">
       <item>
        <widget class="QGroupBox" name="level">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>LEVEL</string>
         </property>
         <widget class="QRadioButton" name="debutant">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>DEBUTANT</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="normal">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>80</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>NORMAL</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mode">
         <property name="title">
          <string>MODE</string>
         </property>
         <widget class="QRadioButton" name="placement">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string>PLACEMENT</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="file">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>80</y>
            <width>91</width>
            <height>22</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>FILE</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>941</width>
      <height>771</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>2000</width>
      <height>2000</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressource.qrc">:/ressources/unknown.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>90</y>
      <width>928</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="battleField" rowstretch="0" columnstretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QLabel" name="backgroundbattle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>931</width>
      <height>561</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>backgroundbattle</zorder>
   <zorder>background</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>gridLayoutWidget_2</zorder>
   <zorder>gridLayoutWidget_3</zorder>
   <zorder>groupBox</zorder>
  </widget>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
